### ISSUE TITLE
panic: runtime error: invalid memory address or nil pointer dereference

### ISSUE THREAD

**Issue by goncalossilva** [github/git-lfs#758]
I'm trying to set up Git LFS to track a couple of directories (and everything inside them) that contain multiple big files.

`.gitattrubutes` contains:

```
FolderA/**/* filter=lfs diff=lfs merge=lfs -text
FolderB/**/* filter=lfs diff=lfs merge=lfs -text
```

When I run `git add .` (which adds those folders, previously untracked) and then I run `git lfs status`, I get the following:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x8 pc=0x5b867]

goroutine 7 [running]:
github.com/github/git-lfs/lfs.revListIndex.func1(0xc82000b380, 0xc820015ec0, 0xc82001b080)
	/private/tmp/git-lfs20151001-17148-1qfl3ns/git-lfs-1.0.0/src/github.com/github/git-lfs/lfs/scanner.go:300 +0x493
created by github.com/github/git-lfs/lfs.revListIndex
	/private/tmp/git-lfs20151001-17148-1qfl3ns/git-lfs-1.0.0/src/github.com/github/git-lfs/lfs/scanner.go:305 +0x20c

goroutine 1 [runnable]:
os/exec.(*Cmd).closeDescriptors(0xc820094f00, 0xc820011000, 0x3, 0x4)
	/usr/local/Cellar/go/1.5.1/libexec/src/os/exec/exec.go:239
os/exec.(*Cmd).Start(0xc820094f00, 0x0, 0x0)
	/usr/local/Cellar/go/1.5.1/libexec/src/os/exec/exec.go:335 +0x894
github.com/github/git-lfs/lfs.startCommand(0x489428, 0x3, 0xc8200519e8, 0x2, 0x2, 0xc82000b600, 0x0, 0x0)
	/private/tmp/git-lfs20151001-17148-1qfl3ns/git-lfs-1.0.0/src/github.com/github/git-lfs/lfs/scanner.go:433 +0x325
github.com/github/git-lfs/lfs.catFileBatch(0xc82001b200, 0xc82001b200, 0x0, 0x0)
	/private/tmp/git-lfs20151001-17148-1qfl3ns/git-lfs-1.0.0/src/github.com/github/git-lfs/lfs/scanner.go:359 +0xa6
github.com/github/git-lfs/lfs.ScanIndex(0x0, 0x0, 0x0, 0x0, 0x0)
	/private/tmp/git-lfs20151001-17148-1qfl3ns/git-lfs-1.0.0/src/github.com/github/git-lfs/lfs/scanner.go:187 +0x290
github.com/github/git-lfs/commands.statusCommand(0x6b2060, 0x6daf30, 0x0, 0x0)
	/private/tmp/git-lfs20151001-17148-1qfl3ns/git-lfs-1.0.0/src/github.com/github/git-lfs/commands/command_status.go:27 +0xbd
github.com/github/git-lfs/vendor/_nuts/github.com/spf13/cobra.(*Command).execute(0x6b2060, 0x6daf30, 0x0, 0x0, 0x0, 0x0)
	/private/tmp/git-lfs20151001-17148-1qfl3ns/git-lfs-1.0.0/src/github.com/github/git-lfs/vendor/_nuts/github.com/spf13/cobra/command.go:477 +0x403
github.com/github/git-lfs/vendor/_nuts/github.com/spf13/cobra.(*Command).Execute(0x6b2bc0, 0x0, 0x0)
	/private/tmp/git-lfs20151001-17148-1qfl3ns/git-lfs-1.0.0/src/github.com/github/git-lfs/vendor/_nuts/github.com/spf13/cobra/command.go:551 +0x46a
github.com/github/git-lfs/commands.Run()
	/private/tmp/git-lfs20151001-17148-1qfl3ns/git-lfs-1.0.0/src/github.com/github/git-lfs/commands/commands.go:88 +0x23
main.main()
	/private/tmp/git-lfs20151001-17148-1qfl3ns/git-lfs-1.0.0/git-lfs.go:9 +0x14

goroutine 17 [syscall, locked to thread]:
runtime.goexit()
	/usr/local/Cellar/go/1.5.1/libexec/src/runtime/asm_amd64.s:1696 +0x1

goroutine 6 [runnable]:
github.com/github/git-lfs/lfs.revListIndex.func1(0xc82000b120, 0xc820015ec0, 0xc82001af60)
	/private/tmp/git-lfs20151001-17148-1qfl3ns/git-lfs-1.0.0/src/github.com/github/git-lfs/lfs/scanner.go:300 +0x493
created by github.com/github/git-lfs/lfs.revListIndex
	/private/tmp/git-lfs20151001-17148-1qfl3ns/git-lfs-1.0.0/src/github.com/github/git-lfs/lfs/scanner.go:305 +0x20c

goroutine 8 [chan send]:
github.com/github/git-lfs/lfs.ScanIndex.func2(0xc82001af60, 0xc82001b0e0, 0xc82001b080)
	/private/tmp/git-lfs20151001-17148-1qfl3ns/git-lfs-1.0.0/src/github.com/github/git-lfs/lfs/scanner.go:171 +0x19c
created by github.com/github/git-lfs/lfs.ScanIndex
	/private/tmp/git-lfs20151001-17148-1qfl3ns/git-lfs-1.0.0/src/github.com/github/git-lfs/lfs/scanner.go:180 +0x219

goroutine 9 [syscall]:
syscall.Syscall(0x3, 0x6, 0xc8200e8000, 0x4000, 0x0, 0x0, 0x0)
	/usr/local/Cellar/go/1.5.1/libexec/src/syscall/asm_darwin_amd64.s:16 +0x5
syscall.read(0x6, 0xc8200e8000, 0x4000, 0x4000, 0x0, 0x0, 0x0)
	/usr/local/Cellar/go/1.5.1/libexec/src/syscall/zsyscall_darwin_amd64.go:972 +0x5f
syscall.Read(0x6, 0xc8200e8000, 0x4000, 0x4000, 0x0, 0x0, 0x0)
	/usr/local/Cellar/go/1.5.1/libexec/src/syscall/syscall_unix.go:160 +0x4d
os.(*File).read(0xc820030150, 0xc8200e8000, 0x4000, 0x4000, 0x0, 0x0, 0x0)
	/usr/local/Cellar/go/1.5.1/libexec/src/os/file_unix.go:211 +0x75
os.(*File).Read(0xc820030150, 0xc8200e8000, 0x4000, 0x4000, 0x0, 0x0, 0x0)
	/usr/local/Cellar/go/1.5.1/libexec/src/os/file.go:95 +0x8a
bufio.(*Reader).fill(0xc82001b1a0)
	/usr/local/Cellar/go/1.5.1/libexec/src/bufio/bufio.go:97 +0x1e9
bufio.(*Reader).Read(0xc82001b1a0, 0xc8200ec000, 0x1000, 0x1000, 0x0, 0x0, 0x0)
	/usr/local/Cellar/go/1.5.1/libexec/src/bufio/bufio.go:207 +0x260
bufio.(*Scanner).Scan(0xc8200d2f20, 0x1000)
	/usr/local/Cellar/go/1.5.1/libexec/src/bufio/scan.go:180 +0x877
github.com/github/git-lfs/lfs.catFileBatchCheck.func1(0xc82000b600, 0xc82001b200)
	/private/tmp/git-lfs20151001-17148-1qfl3ns/git-lfs-1.0.0/src/github.com/github/git-lfs/lfs/scanner.go:325 +0x101
created by github.com/github/git-lfs/lfs.catFileBatchCheck
	/private/tmp/git-lfs20151001-17148-1qfl3ns/git-lfs-1.0.0/src/github.com/github/git-lfs/lfs/scanner.go:342 +0x138

goroutine 10 [runnable]:
github.com/github/git-lfs/lfs.catFileBatchCheck.func2(0xc82001b0e0, 0xc82000b600)
	/private/tmp/git-lfs20151001-17148-1qfl3ns/git-lfs-1.0.0/src/github.com/github/git-lfs/lfs/scanner.go:345 +0x65
created by github.com/github/git-lfs/lfs.catFileBatchCheck
	/private/tmp/git-lfs20151001-17148-1qfl3ns/git-lfs-1.0.0/src/github.com/github/git-lfs/lfs/scanner.go:349 +0x167
```

Any ideas?

**Comment by goncalossilva** [comment]
Hum, running the command right after this happened works as expected. This can be closed, although I'm leaving the stacktrace as it can reveal some hidden issue.

**Comment by technoweenie** [comment]
Hi, I'm going back through old issues I missed. This issue was fixed in #801, available in Git LFS v1.0.2+
